content/en/users/tutorials/filetransfer-gridstorage/_index.md

```diff
@@ -0,0 +1,10 @@
+---
+title: "File Transfer with grid storage"
+linkTitle: "File Transfer with grid storage"
+type: docs
+weight: 80
+description: >
+  Use EGI File Transfer to handle data in grid storage
```

Comment:
Are we abusing "EGI"? also "grid storage" is not something we do actually have

Reply:
Is `draft` missing here?

Reply:
Well, the official name of the services includes EGI, right?

Reply:
Added draft flag. 
As discussed in the Monday's meeting, we could make this an example on how to connect FT to S3-compatible object storage.